crates/client/vendor/github.com/containerd/containerd/api/services/events/v1/events.proto

```diff
@@ -45,7 +45,7 @@ service Events {
 	//
 	// Unlike many other methods in containerd, subscribers will get messages
 	// from all namespaces unless otherwise specified. If this is not desired,
-	// a filter can be provided in the format 'namespace==<namespace>' to
+	// a filter can be provided in the format 'namespace=={namespace}' to
```

Comment:
Should we change this upstream first (or just in parallel)? README states this is just a copy paste and not some tooling doing this, but better to keep these in sync even if it's just a comment touch up.